### PR TITLE
Added the description for math.deg()

### LINKED ISSUE #3391: [Term Entry] Lua Math .deg()
### Reason/inspiration (optional)

We would like a new term entry on the `.deg()` method for Lua: [Math](https://www.codecademy.com/resources/docs/lua/mathematical-library). The entry should go in a new file under `docs/content/lua/concepts/mathematical-library/terms/deg/deg.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```lua `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by bcsamrudh**
Closes #3391 

### Description

Added a description about the math.deg() function in lua with examples.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/deg/deg.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Syntax
+
+```lua
+
+math.deg(x)
+
+```
+
+## Example
+
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:41]: There should be a short sentence introducing the example

File: content/lua/concepts/mathematical-library/terms/deg/deg.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```lua
+
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:43]: remove empty lines at the top and bottom of the block

File: content/lua/concepts/mathematical-library/terms/deg/deg.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+local degrees =  math.deg(radians)
+
+print("The angle in degrees :",degrees)
+
+```
+
+The code will return the following output:
+
+```shell
+
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:57]: remove empty lines

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@bcsamrudh I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/lua/concepts/mathematical-library/terms/deg/deg.md (hunk 3 above)
Comment (bcsamrudh) [reply content/lua/concepts/mathematical-library/terms/deg/deg.md:48]: The lint check fails, if I do that.

File: content/lua/concepts/mathematical-library/terms/deg/deg.md (hunk 2 above)
Comment (bcsamrudh) [reply content/lua/concepts/mathematical-library/terms/deg/deg.md:34]: The lint check fails, if I do that.

**Review by caupolicandiaz — COMMENTED** [review]
@bcsamrudh The metadata is still incorrect. Reference another entry to ensure you have the correct spacing, indentations, etc. And there should be no empty lines. Also remove the empty lines previously noted within the code blocks.

**Comment by caupolicandiaz** [comment]
@bcsamrudh I'm closing this due to inactivity

**Pull request closed without merging**